### DISCUSSION TITLE
Include SELECT2_THEME settings

### DISCUSSION THREAD

**Discussion by CleitonDeLima** [codingjoe/django-select2#99] [Ideas]
Select2 accepts [themes](https://select2.org/appearance#themes), an example is [bootstrap4](https://github.com/ttskch/select2-bootstrap4-theme).

To configure it, you need to include a new css file and add the theme option when starting select2.
```html
<link rel="stylesheet" href="/path/to/select2.css">
<link rel="stylesheet" href="/path/to/select2-bootstrap4.min.css">
```
```js
$('select').select2({
    theme: 'bootstrap4',
});
```

The settings could look like this:
```python
# accept str too
SELECT2_CSS = (
    "/path/to/select2.css",
    "/path/to/select2-bootstrap4.min.css",
)
SELECT2_THEME = "bootstrap4"
```

I am willing to develop this option if it is valid.

**Comment by codingjoe** [comment]
Excellent suggestion @CleitonDeLima. I escalated this into an issue. Would you be interested in providing a pull-request for this?
https://github.com/codingjoe/django-select2/issues/101

**Reply by CleitonDeLima** [reply]
Of course, i can do the PR.

**Reply by codingjoe** [reply]
It's been released in 7.9.0 🎉
